maestro-cli/src/main/java/maestro/cli/runner/TestRunner.kt

```diff
@@ -88,11 +117,16 @@ object TestRunner {
 
                         runCatching(resultView, maestro) {
                             MaestroCommandRunner.runCommands(
-                                maestro,
-                                device,
-                                resultView,
-                                commands,
-                                FlowDebugMetadata()
+                                maestro = maestro,
+                                device = device,
+                                view = resultView,
+                                commands = commands,
+                                debugOutput = FlowDebugOutput(),
+                                // TODO(bartekpacia): make AI outputs work in continuous mode
```

Comment:
Better to create issue as well after release!

Reply:
Created https://github.com/mobile-dev-inc/maestro/issues/1972 to track this